README.md

```diff
@@ -1,2 +1,20 @@
-# HW01
-Edit the readme
+HW01
+---
+Kelly O'Leary
+
+7/5/20
+
+### About me
+
+I'm an incoming student in the BMB. I enjoy making music, growing plants, and being outdoors in my freetime. For the past couple years I've been a member of the [Kossiakoff lab](https://kosslab.uchicago.edu/) here at UChicago.
+
+### States I've lived in
+* Wisconsin
+* Minnesota
+* Montana (_my favorite_)
+* Colorado
+* Illinois
+
+### What do I want to study?
+I am interested in the use of protein engineering to create molecular tools. For example, synthetic antibodies combined with various cargo "plugins" can be used all the way from therapeutics to structural chaperones. 
```

Comment:
I am also an incoming student in the BMB program and interested in protein engineering!